packages/zoe/exported.js

```diff
@@ -1,11 +1 @@
-// @jessie-check
-
-import './src/contractFacet/types-ambient.js';
-import './src/zoeService/types-ambient.js';
-import './src/contractSupport/types-ambient.js';
-import './src/contracts/exported.js';
-import './src/types-ambient.js';
-import '@agoric/notifier/exported.js';
-import '@agoric/ertp/exported.js';
-import '@agoric/store/exported.js';
-import '@agoric/swingset-vat/exported.js';
+// Dummy file for .d.ts twin to declare ambients
```

Comment:
That seems like a cheap trick, what's the drawback of doing that everywhere?

Reply:
Do you mean carrying out the best practice written here? https://github.com/Agoric/agoric-sdk/blob/master/docs/typescript.md#dts-modules

Or something more specific? If it's about `exported.js`, I'd like to move away from that pattern altogether. Zoe is just recalcitrant